runtimes/nodejs/object-delete.js

```diff
@@ -6,14 +6,29 @@
  *
  * In this case, the args variable will look like:
  *   {
- *     "Bucket": "your COS bucket name",
- *     "Key": "Name of the object to delete"
+ *     "bucket": "your COS bucket name",
+ *     "key": "Name of the object to delete"
  *   }
  */
-var CloudObjectStorage = require('ibm-cos-sdk');
-function main(args) {
-  let { cos, params } = getParamsCOS(args, CloudObjectStorage);
-  return cos.deleteObject(params).promise();
+const CloudObjectStorage = require('ibm-cos-sdk');
+
+async function main(args) {
+  const { cos, params } = getParamsCOS(args, CloudObjectStorage);
+
+  let response;
+  const result = {
```

Comment:
Are these instances really `const` since they are being changed later on?

Reply:
from js reference:
The const declaration creates a read-only reference to a value. It does not mean the value it holds is immutable, just that the variable identifier cannot be reassigned. For instance, in the case where the content is an object, this means the object's contents (e.g., its parameters) can be altered.